spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -259,7 +259,7 @@ describe('document-capture/components/acuant-capture', () => {
       expect(window.AcuantCameraUI.end.called).to.be.false();
     });
 
-    it('shows error if capture fails', async () => {
+    it('shows error if capture fails: legacy version of Acuant SDK', async () => {
       const trackEvent = sinon.spy();
```

Comment:
Can you clarify how this is selecting the legacy version? I'm not seeing version number differences in the two specs.

Reply:
If we look at line 275, we can see that the second argument for the start function is the `onFailure` callback.  If this were the more recent version of Acuant SDK, the `onFailure` callback would be included in the first argument for the start function, which are the `AcuantCameraUICallbacks`.

To give a more general answer, I decided to test that both version 11.8.2 and version 11.9.1 are supported by stubbing the `start` function as it would be called for the two different versions.  I didn't put any direct references to version number in the specs.